### PR TITLE
first implement

### PR DESCRIPTION
**Pull request by ghodsehimanshu17295665**
1] created Models - TimeStampedModel, User, Task, Comment
2] Register models in admin file
3] created Form - SignupForms, LoginForms, TaskForm, CommentForm
4] created Views - HomeView, SignupView, LoginView, LogoutView
5] created Templates file

### CODE REVIEW

File: app/forms.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+            "title",
+            "description",
+            "status",
+            "priority",
+            "assignee",
+            "due_date",
+        ]
+
+
+class CommentForm(forms.ModelForm):
```
Comment (tanishgupta905) [inline app/forms.py:44]: what is the use of this Model ?

**Review by tanishgupta905 — APPROVED** [review]

**Comment by tanishgupta905** [comment]
fix flake8 in your code

File: app/forms.py (hunk 1 above)
Comment (ghodsehimanshu17295665) [reply app/forms.py:44]: Sir, TaskForm and CommentForm are used to create or update record in the databaseModel 
and i'm use the Django Model Form so it is direct links the form to specific model. these allows django automatically generated form fields based on the fields defined in the model.

File: app/models.py [hunk 2]
```diff
@@ -0,0 +1,65 @@
+from django.contrib.auth.models import AbstractUser
+from django.db import models
+from django.utils import timezone
+
+
+class TimeStampedModel(models.Model):
```
Comment (rajatShr) [inline app/models.py:6]: No need to create the TimeStampedModel, just import this
from django_extensions.db.models import TimeStampedModel

**Review by rajatShr — APPROVED** [review]

File: app/models.py (hunk 2 above)
Comment (ghodsehimanshu17295665) [reply app/models.py:6]: Okay Sir.

**Review by tanishgupta905 — APPROVED** [review]

**Pull request merged**